pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
-                                words_list = corrected_text.lower().split("_")
-                                corrected_text = f"{words_list[0].capitalize()}_{'_'.join(map(str.upper, words_list[1:]))}"
-                                URL = "https://tarkov-market.com/item/"+corrected_text
+            (cnts, _) = cv2.findContours(edged.copy(), cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
+
+            idx = 0
+            imagesList = []
+            areaList = []
+            i = 0
+            for c in cnts:
+                ## For testing draw the contours
+                peri = cv2.arcLength(c, True)
```

Ruff (0.8.0)

Boolean positional value in function call

(FBT003)